### ISSUE TITLE
KWIC: Sidebar hidden and shown again when switching KWIC pages

### ISSUE THREAD

**Issue by janiemi** [spraakbanken/korp-frontend#227]
When the sidebar is visible and you change to another KWIC page, the sidebar is first hidden and then restored when the new KWIC page is shown. Is that intentional, for example, to show that Korp is doing something?

Personally, I find the sidebar sliding back and forth unnecessary, so I’m wondering if it would make sense to keep the state of the sidebar across page changes: if the sidebar is visible, do not hide it, and if it’s hidden, do not show it. And would it also make sense to show the sidebar for a new page on the context view if it was visible on the previous page?

**Comment by majsan** [comment]
I agree! It should stay in the same state as it was before on page change. Same for the context view I guess?

**Comment by janiemi** [comment]
Good. I think it could or should be the same for the context view: if the user has clicked on a word in the context view so that the sidebar is shown, I think they might wish it to be shown after a page change, too.

Should the initial state after a search or when switching views still be the same as present: for the KWIC view, the first matched word of the first hit selected and the sidebar shown, and for the context view, no word selected and the sidebar hidden?

I already began trying this out when investigating #201, but I decided that it’s better to ask first. And it might not be as simple as I first thought to get everything right.

**Comment by majsan** [comment]
I think the initial states should be the same. It's probably a good assumption that users who use the context view wants to read the text and not see attributes. But, if it complicates the code a lot, it is not worth it.

**Comment by janiemi** [comment]
I agree. I’ll see if I can make it work with reasonable effort, but I have some more urgent tasks at hand right now.
